blockstore/badger/blockstore.go

```diff
@@ -627,7 +656,18 @@ func (b *Blockstore) Flush(context.Context) error {
 	b.lockDB()
 	defer b.unlockDB()
 
-	return b.db.Sync()
+	// fsync the new db first
+	if b.dbNext != nil {
+		if err := b.dbNext.Sync(); err != nil {
+			return err
```

Comment:
should we at least continue and try and sync the db this was actually called on? perhaps we collect this error and return a multierr if we get one from both

Reply:
`go.uber.org/multierr` is used more often and it'd be my preferred option, so just a `multierr.Combine(err1, err2)` should be fine here, and it'll play fine with `nil`s too.